packages/functions/src/common/BookAppointmentHelper.ts

```diff
@@ -47,7 +47,7 @@ export async function bookAppointment(
   appointmentToBook.lastChangeType = BookingChange.BOOKED;
   appointmentToBook.status = AppointmentStatus.BOOKED;
 
-  if ( coordinatorId && donorId === MANUAL_DONOR_ID) {
+  if (donorId === MANUAL_DONOR_ID) {
```

Comment:
I suggest you still check here that coordinatorId is not undefined

Reply:
done